src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
ah I see. Perhaps the Materializer is only needed because I'm testing the templates via a controller? I will see what happens if I get rid of the controller and call the templates directly from the test like InputRadioGroupSpec does